api/plugin-api.cpp

```diff
@@ -5,15 +5,13 @@
 namespace plugin_api::detail {
 
 BaseDialog::BaseDialog() = default;
-void BaseDialog::closeEvent(QCloseEvent*)
+
+void BaseDialog::closeEvent(QCloseEvent* aEvent)
 {
-    if (isVisible())
-    {
-        hide();
```

Comment:
These two lines can't be removed due to other core functionality depending on it.